packages/apktool.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,22 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolLnk = 'apktool.lnk'
```

Comment:
In most packages we define `$toolName` and `$shortcut = Join-Path $shortcutDir "$toolName.lnk"` (it is also [documented](https://github.com/mandiant/VM-Packages/wiki/Contributing#best-practices)).  I think keeping packages as similar as possible makes it easier to read and maintain.